test/unit/routerTest.js

```diff
@@ -63,6 +63,17 @@ describe('HTTP Router', () => {
         ctx.response.header.should.be.ok
       })
 
+      it('should route an incomming http request and then stream the response into gridfs', async () => {
```

Comment:
Typo: `incomming`

Reply:
fixed

Reply:
commit fixing this - https://github.com/jembi/openhim-core-js/pull/1036/commits/7afbbafb09eb82730888261e1ef66c9228120fc6